src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames';
+import { TodoType } from '../../types/TodoType';
+
+export enum SortType {
```

Comment:
maybe it would be better to rename it? because it seems to be filter not sort
```suggestion
export enum FilterType {
```